src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/update/SetOperation.java

```diff
@@ -28,7 +28,7 @@ public static SetOperation construct(ObjectNode args) {
   }
 
   @Override
-  public boolean updateDocument(ObjectNode doc) {
+  public boolean updateDocument(ObjectNode doc, UpdateTargetLocator targetLocator) {
```

Comment:
Set operation also need to support dot notation, Is this PR only for unset?
Eg: 
`db.products.updateOne(
{ _id: 100 },
{ $set: { "details.make": "Kustom Kidz" } }
)`

Reply:
Correct: this is only for $unset, #166 is the issue for $set and I'll work on that next.